src/core/sm/smclient/itf/smclient.hpp

```diff
@@ -24,7 +24,7 @@ namespace aos::sm::smclient {
  */
 class SMClientItf : public alerts::SenderItf,
                     public monitoring::SenderItf,
-                    public logging::SenderItf,
+                    public aos::logging::SenderItf,
```

Comment:
It is strange that only logging causes the issue. alerts::SenderItf also located in core common.